__tests__/identity.test.js

```diff
@@ -1,6 +1,13 @@
 const { v4: uuidv4 } = require('uuid')
-const { apiUrl, idRealmName, idAppName, clientRegistrationToken } = global
+const {
+  apiUrl,
+  idRealmName,
+  idAppName,
+  clientRegistrationToken,
+  testTozIDGroupName,
+} = global
 const Tozny = require('../node')
+const { AccessRequest, AccessRequestSearchRequest } = require('../types')
```

Comment:
i think `AccessRequestSearchRequest` is unused

Reply:
you were righ!